code/game/machinery/suit_storage_unit.dm

```diff
@@ -517,3 +524,27 @@
 		I.play_tool_sound(src, 50)
 		visible_message("<span class='notice'>[usr] pries open \the [src].</span>", "<span class='notice'>You pry open \the [src].</span>")
 		open_machine()
+
+// Mapping helper unit takes whatever lies on top of it
+/obj/machinery/suit_storage_unit/inherit/Initialize(mapload)
+	. = ..()
+	wires = new /datum/wires/suit_storage_unit(src)
+	if(mapload)
+		addtimer(CALLBACK(src, .proc/inherit_contents), 0)
```

Comment:
Should probably move this to LateInitialize, make sure that the original initialize here returns the lateinit hint

Reply:
Borrowed the method from `/obj/structure/closet/Initialize(mapload)` but I can certainly use that instead.